pennylane/plugins/default_qubit_autograd.py

```diff
@@ -15,13 +15,7 @@
 reference plugin.
 """
 from pennylane.operation import DiagonalOperation
-
-try:
-    from pennylane import numpy as np
-
-except ImportError as e:

```

Comment:
decided to do away with it in the end?

Reply:
I can't get codecov to work, it's been driving me mad all day! So this is an attempt for codecov to give me the tick 😝

Reply:
+1 for codecov problems!
I was waiting ages for a commit to master so that I could update https://github.com/PennyLaneAI/pennylane/pull/716 and get the tick!